src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/StandardizedCompetencyDTO.java

```diff
@@ -9,7 +9,7 @@
  * DTO containing {@link StandardizedCompetency} data. It only contains the id of the knowledge area and source.
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Also not really related to PR but just to be sure that unknown properties sent from the client are ignored (and it does not fail), one could add `@JsonIgnoreProperties(ignoreUnknown = true)`